Documentation/project-docs/linux-performance-tracing.md

```diff
@@ -161,7 +161,7 @@ With that installed download the symbols to a local directory.  if your installe
 2.1.0 the command to do this is 
```

Comment:
There is "perfollect" typo on the previous line - could you please fix it as well while you are on it?